### ISSUE TITLE
ValueError: Expected the value in a schema to be a string, but got: <class 'swagger_to.swagger.RawDict'>

### ISSUE THREAD

**Issue by stevenxuwoss** [Parquery/swagger-to#132]
if definitions properties has name with description, will cause this issue
```
  DeviceDefineInfoRes:
    type: object
    discriminator: ''
    properties:
      createPerson:
        type: string
      createTime:
        type: string
      description:
        type: string
      id:
        type: string
      manufacturer:
        type: string
      name:
        type: string
      number:
        type: string
      productTypeCode:
        type: integer
        format: int32
      updatePerson:
        type: string
      updateTime:
        type: string
    description: ''
```
<img width="729" alt="20211027104911" src="https://user-images.githubusercontent.com/17000812/138991549-b512b07d-f789-4df9-8436-92e87c698d4c.png">

*(@mristin, 2021-10-27: added markers for multi-line code)*

**Comment by mristin** [comment]
@stevenxuwoss I just realized I haven't looked at this issue. Is it still critical to you?

**Comment by nicolasassi** [comment]
@mristin I had the same problem sadly.

**Comment by mristin** [comment]
@nicolasassi thanks for letting me know! I'll try to have a look at this tomorrow (Monday).

**Comment by mristin** [comment]
Hi @nicolasassi ,
Unfortunately, I got swamped with work & mixed in holidays, so I didn't have time to look into this problem. At the moment, I do not have time to fix this issue.

However, if you'd like to make a pull request, I'd be more than glad to support you, introduce you to the code and review your changes.

**Comment by salmanee** [comment]
Also running into the same issue

**Comment by mristin** [comment]
@salmanee could you please post a minimal schema that causes the error?

**Comment by salmanee** [comment]
@mristin This is a small snippet.

**Comment by AkuPython** [comment]
@mristin Apologies for the mess above, it's my first PR. Next time I won't include the Issue # in my Fork commits...

Essentially, the {}["description"]: value is str... (If required) should be _below_ the other checks. The schema _validation_ accurately analyses if "description" must be a string, and errors as required.
Here, checking if the "description" value is a string only appears to be needed to trigger .strip() for documentation, not to determine the validity of the "description" values type.

**Comment by mristin** [comment]
@AkuPython thanks a lot again for your fix! I merged it in and released the patch version 5.0.1.

Please re-open if the issue re-surfaces.
